queue/queueController.js

```diff
@@ -29,7 +29,8 @@ define('queueController', [], function() {
 						appointment.preparing = appointment.timeAppointmentStarted != null;
 						appointment.ended = appointment.timeAppointmentEnded != null;
 						appointment.name = appointment.firstName + " " + appointment.lastName;
-						appointment.noshow = appointment.noshow == true; // Do this since the SQL returns 0/1 for false/true, and we want it to be true/false instead of 0/1
+						appointment.noShow = appointment.noShow == true; // Do this since the SQL returns 0/1 for false/true, and we want it to be true/false instead of 0/1
```

Comment:
I've confused about what this is doing. This seems like it shouldn't do anything, but apparently it doesn't work without it so 🤷‍♂️